src/fetching/deleteEventResponse.ts

```diff
@@ -1,17 +1,29 @@
 import apiUrl from '../utils/apiUrl';
 
-import { ZetkinEventSignup, ZetkinMembership } from '../types/zetkin';
+import { ZetkinMembership } from '../types/zetkin';
 
-export default async function deleteEventResponse({ eventId, orgId } : ZetkinEventSignup) : Promise<void> {
-    const mUrl = apiUrl('/users/me/memberships');
-    const mRes = await fetch(mUrl);
-    const mData = await mRes.json();
-    const orgMembership = mData.data.find((m : ZetkinMembership) => m.organization.id === orgId);
+function defaultFetch(path : string, init? : RequestInit) {
+    const url = apiUrl(path);
+    return fetch(url, init);
+}
```

Comment:
This reoccurs in many locations, and should probably moved to a shared module and reused. Perhaps in `src/fetching/index.ts`?

Reply:
Agree!